### PR TITLE
deps: update deps version

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by jtyjty99999** [comment]
appveyor这个windows环境有点醉啊

File: appveyor.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
-  - npm i npminstall && node_modules\.bin\npminstall
+  - npm i npminstall && node_modules\.bin\npminstall --force
```
Comment (fengmk2) [inline appveyor.yml]: 呃，npminstall 没有 --force 这个参数的。。

Comment (atian25) [inline appveyor.yml]: 我刚发上来实验的... 原来不支持啊, 本地没报错, 还以为是文档没写.

Comment (atian25) [inline appveyor.yml]: 先干掉了.

**Comment by popomore** [comment]
+1

**Pull request merged**